lib/timetable-form/Form.tsx

```diff
@@ -1,38 +1,62 @@
 import { useState } from "react";
 import styles from "./styles.module.css";
-
-const stageNames = [
-  "Timetable published",
-  "Draft plan for public consultation published",
-  "Public consultation start",
-  "Public consultation end",
-  "Submit plan for examination",
-  "Examination hearing start",
-  "Planning inspectorate examination start",
-  "Planning inspectorate examination end",
-  "Planning inspectorate found sound",
-  "Examination hearing end",
-  "Inspector report published",
-  "Plan adopted",
-] as const;
-
-type StageName = (typeof stageNames)[number];
+import { DevelopmentPlanTimetable, Stages } from "../types/timetable";
+import { stageNames } from "../constants";
 
 const defaultDate = "";
 
-type Stages = {
-  [key in StageName]: string;
-};
-
 const initialStages = Object.fromEntries(
   stageNames.map((stage) => [stage, defaultDate])
 ) as Stages;
 
+type FormData = {
+  LPA: string;
+  stages: Stages;
+};
+
+const formStateToDevelopmentPlanTimetables = (
+  state: FormData
+): DevelopmentPlanTimetable[] => {
+  return Object.entries(state.stages).map(([event, eventDate]) => ({
+    reference: "XXX-XXX-XXX",
+    name: "",
+    developmentPlan: "dorcester-new-local-plan",
+    developmentPlanEvent: event,
+    eventDate: eventDate,
+    notes: "",
+    organisation: state.LPA,
+    entryDate: new Date().toISOString(),
+    startDate: new Date().toISOString(),
+  }));
+};
+
+const DevPlanToCSVString = (timeTables: DevelopmentPlanTimetable[]): string => {
```

Comment:
Very minor stylistic point but why does this function name start with a capital letter?